src/main/resources/application-local.yaml

```diff
@@ -0,0 +1,14 @@
+aws:
+  sqs:
+    config:
+      host: localhost
+      port: 4566
+      url: http://${aws.sqs.config.host}:${aws.sqs.config.port}
+    audit:
+      url: ${aws.sqs.config.url}/queue/${aws.queue.audit.name}
+
+spring:
```

Comment:
is this -local config? shouldn't the only difference be the localhost or real URL in DB host?
the queue URL above has to be different but not the postgres?

Reply:
This has been changed 👍🏽